### ISSUE TITLE
Task 22 (main sidebar menu)

### ISSUE THREAD

**Issue by pmvald** [Appers-lab/billihomes-frontend#22]
(Make sure you have read the mobile design guide found in doc/mobile-guide.md)

We want to design a sidebar menu similar to mobilekit (when you click on the toggle button)

We don't need to make its sliding animation, etc, only the box itself. Use the same width as Mobilekit does. The menu has to have these items:

- How it works
- Mortgage application
- About us
- Contact us
- section: Properties (this is a section, like section "friends" that you see on mobilekit)
- Condos
- Townhouses
- Houses
- Lands
- section: my account
- My appointments
- My profile
- Login
- Logout

 We already have some materials for this menu:
- the header is designed in task 24. There are two types of the header: 1. when user is logged in, 2- when user i logged out. In the first case, for the header title show a welcome message, like "Welcome John Doe" (no need for avatar), and in the second case we show the title "Welcome guest" and the menu menu contains a login link (in addition to the close button). You an use an icon for the login link
- The menu items are designed in Task 19 (listviews)
- The section is designed in Task 25 (Hmm, ... the section may need to be made more general. Currently it has some background color and its font size is big. Maybe we should make the bakcground transparent and create modifier --font-lg to make the text big as it is now)

So basically your task would be to put all these components together. Perhaps the only specific stylw you need to apply to the sidebar is "width:80%" which for now just hardcode it with style property. I don't think that we need to define any new class for the sidebar. Also pick icons/images for the menu items as well.

**Comment by pmvald** [comment]
@j-ae Now putting all the materials together you can create the sidebar menu. You may need to make changes to the section class though:
1. make the background transparent
2. The current font-size for the section is large. You can make it smaller (same size as the section header in the mobilekit sidebar) and introduce the modifier --font-lg as an option to make the text big. For I noticed most section headers are rather normal size than large size. 

Use css variables of your choice.

**Comment by j-ae** [comment]
@pmvald, Hi, I have two small questions.

![v2 (2)](https://user-images.githubusercontent.com/44935193/122987283-4815c800-d355-11eb-8b1c-3debd9030dfc.jpg)
![v1 (3)](https://user-images.githubusercontent.com/44935193/122985635-4519d800-d353-11eb-9752-232ffc972335.jpg)

**1. Should I remove the indentation of list items?**
In Mobilekit, the section header and the list items have the same vertical alignment.

**2. Should I remove the border line in the section header?**
There are top and bottom border lines in the section header. Should I add a modifier(--no-border) to remove them?

**Comment by pmvald** [comment]
@j-ae To solve this problem (and other similar ones) we can defined a global scss variable `$padding-x-base` and ensure that all our components have the same left/right padding equal to this variable value. We may need to make changes to the `s-listview` class to enforce this restriction, and same with the header. I think setting `$padding-x-base` to `$spacer` should be good (or `0.5*$spacer`? see which one makes it look nicer)

**Comment by pmvald** [comment]
@j-ae We can define a mixin called "container" (in `styles/_mixins.m.scss`) which adds the padding-x equal to `$padding-x-base`. We use this mixin for all the top-level components to make all align with each other.
